### PR TITLE
MM-28113 - Frontend UI updates

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
https://mattermost.atlassian.net/browse/MM-28113

- [x] Missing remove optional description for task
- [x] Clicking either the title or pencil icon can edit the title
- [x] Stage and steps should be left-aligned rather than indented
- [x] Section heading says “Stages and steps”, should be “Stages and tasks”
- [x] Add Image to assigned item in Task
- [x] Move run button to the left 
- [x] Clicking the empty area to the right of step description also clicks the button
- [x] Clicking the empty area to the right of steps deletes the step
- [x] RHS missing help text for incident channel
- [x] RHS says “Channel type” rather than “Incident Channel”
- [x] Should have trash icon on hover
- [x] RHS has placeholder image but should be blank
- [x] Playbook background is not grey
- [x] Playbooks first in the header, and then Incidents
- [x] Add Close button in the header for Playbooks area

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28113

#### Screenshots
localhost_8065_mattermost_com mattermost plugin-incident-response_playbooks_wpis6wxy3tb4xfrrcyu1ohjd8e (2)
localhost_8065_mattermost_com mattermost plugin-incident-response_playbooks_wpis6wxy3tb4xfrrcyu1ohjd8e (1)

### CODE REVIEW

**Comment by asaadmahmood** [comment]
@lieut-data Can you refresh, everything there should actually be left aligned.

**Comment by asaadmahmood** [comment]
It should actually look like this.
localhost_8065_mattermost_com mattermost plugin-incident-response_playbooks_wpis6wxy3tb4xfrrcyu1ohjd8e (3)

**Comment by lieut-data** [comment]
Oh, fascinating -- another one of those "old CSS left behind bugs." Yes, working here now!

File: webapp/src/components/backstage/drag_handle.tsx [hunk 1]
```diff
@@ -2,29 +2,49 @@ import React, {FC, useState} from 'react';
...
+        color: var(--center-channel-color);
+    }
 `;
 
 const Content = styled.div`
     flex-grow: 1;
 `;
 
 export interface DragHandleProps {
+    task: boolean;
```
Comment (agarciamontoro) [inline webapp/src/components/backstage/drag_handle.tsx:47]: Can this be step? Just to be consistent with the code.

**Review by agarciamontoro — CHANGES_REQUESTED** [review]
This is great! Thank you for the buttons components, they will come in handy!

I left a minor nit below, and found some old "step" wording here and there:
- `New Step` should be `New Task`  in stage_edit.tsx
- `Step Name` should be `Task Name` in step_edit.tsx
- `Steps` should be `Tasks` in playbook_list.tsx

**Comment by lieut-data** [comment]
All good points, @agarciamontoro -- I've pushed the requested changes.

**Comment by lieut-data** [comment]
@asaadmahmood, I've additionally resolved, assuming that was just to reflect the incident name in the channel header
[x]  Adjust incident inner header to match designs.

image

image

**Review by agarciamontoro — APPROVED** [review]
Neat!

**Pull request merged**